### PR TITLE
Adds documentation for Lupusec alarm control panel

### PR DESCRIPTION
**Pull request by majuss**
**Description:**
This PR add support for the alarm control panel from Lupus Electronics. For now the panel and sensors are supported and tested.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** https://github.com/home-assistant/home-assistant/pull/17691

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/alarm_control_panel.lupusec.markdown
Comment (klaasnicolaas) [inline source/_components/alarm_control_panel.lupusec.markdown]: Use the new configuration variable syntax: [more info](https://developers.home-assistant.io/docs/en/documentation_create_page.html)

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/alarm_control_panel.lupusec.markdown
Comment (frenck) [inline source/_components/alarm_control_panel.lupusec.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/alarm_control_panel.lupusec.markdown
Comment (frenck) [inline source/_components/alarm_control_panel.lupusec.markdown]: Please use all CAPS constant like placeholders. For example, `YOUR_USERNAME`.

File: source/_components/alarm_control_panel.lupusec.markdown
Comment (frenck) [inline source/_components/alarm_control_panel.lupusec.markdown]: Please use all CAPS constant like placeholders. For example, `YOUR_PASSWORD`.

File: source/_components/alarm_control_panel.lupusec.markdown
Comment (frenck) [inline source/_components/alarm_control_panel.lupusec.markdown]: `0.79` has been shipped already some time ago. The next possible release would be `0.81`.

File: source/_components/binary_sensor.lupusec.markdown
Comment (frenck) [inline source/_components/binary_sensor.lupusec.markdown]: `0.79` has been shipped already some time ago. The next possible release would be `0.81`.

File: source/_components/lupusec.markdown
Comment (frenck) [inline source/_components/lupusec.markdown]: `0.79` has been shipped already some time ago. The next possible release would be `0.81`.

File: source/_components/switch.lupusec.markdown
Comment (frenck) [inline source/_components/switch.lupusec.markdown]: `0.79` has been shipped already some time ago. The next possible release would be `0.81`.

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @majuss, thank you for this PR.

I and @klaasnicolaas left you some comments. Could you please take a look? 👍

**Comment by majuss** [comment]
@frenck @klaasnicolaas Thanks for the comments! I will work them off.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @majuss.

Can be merged as soon as the parent PR gets merged.

**Comment by majuss** [comment]
Thank you for the fine tuning!

**Comment by frenck** [comment]
@majuss Your welcome sir! 👍

File: source/_components/lupusec.markdown
Comment (tjorim) [inline source/_components/lupusec.markdown]: Please write Lupusec with a capital letter here,

File: source/_components/lupusec.markdown
Comment (tjorim) [inline source/_components/lupusec.markdown]: here,

File: source/_components/lupusec.markdown
Comment (tjorim) [inline source/_components/lupusec.markdown]: and here as well.

File: source/_components/lupusec.markdown
Comment (tjorim) [inline source/_components/lupusec.markdown]: Isn't the default value 5 seconds? https://github.com/home-assistant/home-assistant/pull/17237/files#diff-e34648182c47cb2ba95be679ce4023b1R23
(I do see that in other places it's 2 so I'm not sure)

File: source/_components/lupusec.markdown
Comment (tjorim) [inline source/_components/lupusec.markdown]: Typo: ressource --> resource (and sentence seems a bit off)

File: source/_components/lupusec.markdown
Comment (tjorim) [inline source/_components/lupusec.markdown]: Remove this empty line

**Review by tjorim — COMMENTED** [review]
Good luck with your first PR, @majuss!
I left some feedback on the documentation.

**Review by tjorim — COMMENTED** [review]
Is "Cloud Push" the correct IoT class? Not Local?

**Comment by majuss** [comment]
@tjorim I think I got everything.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/alarm_control_panel.lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/alarm_control_panel.lupusec.markdown]: 0.83

File: source/_components/alarm_control_panel.lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/alarm_control_panel.lupusec.markdown]: This is a polling integration.

File: source/_components/binary_sensor.lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/binary_sensor.lupusec.markdown]: 0.83

File: source/_components/binary_sensor.lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/binary_sensor.lupusec.markdown]: See above.

File: source/_components/binary_sensor.lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/binary_sensor.lupusec.markdown]: It's confusing to call them switches. Switches in home assistant are actuator devices. The page is for binary sensors.

File: source/_components/lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/lupusec.markdown]: 0.83

File: source/_components/lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/lupusec.markdown]: See above.

File: source/_components/lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/lupusec.markdown]: ... allows the user...

File: source/_components/lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/lupusec.markdown]: There's no downloading or showing the latest image in home assistant?

File: source/_components/lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/lupusec.markdown]: Use two space indentation.

File: source/_components/lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/lupusec.markdown]: This isn't configurable.

File: source/_components/switch.lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/switch.lupusec.markdown]: 0.83

File: source/_components/switch.lupusec.markdown
Comment (MartinHjelmare) [inline source/_components/switch.lupusec.markdown]: See above.

File: source/_components/lupusec.markdown
Comment (MartinHjelmare) [reply source/_components/lupusec.markdown]: `name` config option is missing.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

**Comment by majuss** [comment]
I will work on it asap :) Thanks again @MartinHjelmare for the very helpful comments!

**Review by MartinHjelmare — APPROVED** [review]
Looks good! Can be merged when build passes.

**Pull request merged**
